source/WorkingPractices/reviews.rst

```diff
@@ -70,7 +86,7 @@ Guidance for the SciTech reviewer can be found :ref:`here <scitech_review>`.
 Code and System Review
 ----------------------
 Organising the code reviewer is the responsibility of the developer and is
-done by emailing the :ref:`ssd`. Reviewers are assigned to email requests a
+done by emailing the :ref:`SSD Team <ssd>`. Reviewers are assigned to email requests a
```

Comment:
Organising the <= or => Requesting a - code reviewer ?
I feel that organising is a bit to close to selecting when that's not what we mean.